### PR TITLE
fix: test

### PR DESCRIPTION
**Pull request by joshuarli**
adds a os.walk sorting wrapper to fix test flakes

### CODE REVIEW

File: tests/utils.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+    if topdown:
+        yield f"{top}", sorted(dirs), sorted(nondirs)
+    for name in dirs:
+        new_path = top / name
+        if followlinks or not os.path.islink(new_path):
+            for x in sorted_walk(new_path, topdown, followlinks):
+                yield x
+    if not topdown:
+        yield f"{top}", sorted(dirs), sorted(nondirs)
```
[the remark this answers is not in the document]
Comment (joshuarli) [reply tests/utils.py:31]: huhhhhh that did not occur to me...  good idea!

[the remark this answers is not in the document]
Comment (joshuarli) [reply tests/utils.py:31]: well, the dirs and nondir lists themselves need to also be sorted

[the remark this answers is not in the document]
Comment (asottile-sentry) [reply tests/utils.py:31]: I think ultimately you're just looking for a list of all files created? like `find . -type f` sorta output?

**Review by asottile-sentry — APPROVED** [review]

**Pull request merged**